doc/py_tutorials/py_calib3d/py_multicam_tools/py_multicam_selector.markdown

```diff
@@ -0,0 +1,273 @@
+Multi-camera Calibration Image Selection {#tutorial_py_multicam_image_selector}
+===============================================================================
+
+Goal
+----
+This tool helps you pick good calibration images automatically.
+It works with multiple cameras or a single video.
+
+- Scores frames for **sharpness**, **exposure**, and **pattern coverage**
+- Keeps a small, diverse set that makes calibration more stable
+
+@note The script only **selects images**. It does not calibrate.
+It writes YAML lists that you can feed into calibration later.
+
+> **Intended use:** This tool is designed to pre-filter datasets for later calibration with
+> `samples/cpp/calibration.cpp` or `samples/python/multiview_calibration.py`.
+
+**Source:** `samples/python/multicam_image_selector.py`
+
+Basics
+------
+Calibration works best when the board is sharp, well lit, and appears at varied positions and scales.
+Picking those by hand is slow. This script scans all frames, scores them, and keeps the best ones.
+
+See also:
+- [Camera Calibration](#tutorial_py_calibration)
+- [Camera Calibration and 3D Reconstruction](#tutorial_table_of_content_calib3d)
+
+What the script does
+--------------------
+1. **Read inputs**
+   - A root folder with subfolders per camera (`cam1/`, `cam2/`, …)
+   - Or a video with `--video` (frames sampled by `--video-step`)
+
+2. **Detect the pattern**
+   - Supported: `chessboard`, `circles`, `acircles`, `charuco`, `aruco_grid`
+   - Uses standard OpenCV calls: `cv.findChessboardCorners()`, `cv.cornerSubPix()`, `cv.findCirclesGrid()`, ArUco
+
+3. **Score each frame**
+   - **Sharpness** – Laplacian variance
+   - **Exposure** – checks for under/overexposure
+   - **Coverage** – number and spread of detected points
+   - Final score = weighted sum of these
```

Comment:
As we discussed in the PR with code the expose does not play a lot of role, if the pattern is detected well. If we speak about intrinsics calibration, the first criteria is good spatial coverage. Then,if we have several candidates for each pattern location, it makes sense to select the one with best sharpness, contast, etc.